test/e2e/framework/exec_util.go

```diff
@@ -58,6 +58,7 @@ func (f *Framework) ExecWithOptions(options ExecOptions) (string, string, error)
 
 	const tty = false
 
+	Logf("ExecWithOptions: Clientset creation ")
```

Comment:
nit: Why do we need to add a space at the end of the message after `creation`?

Reply:
/remove-extra-space